### DISCUSSION TITLE
Arduino nano 2040 connect

### DISCUSSION THREAD

**Discussion by L0cutus** [earlephilhower/arduino-pico#162] [Q&A]
Hello,

Any plan to support this board also?
Thanks !

**Comment by earlephilhower** [comment]
While it is possible to use this core on the Arduino branded RP2040, I don't see it as a good idea.

The Arduino board is basically a whole computer on a board with a ton of peripherals (WiFi microcontroller, IMU, etc.) and a hookup to the Arduino IoT cloud.  A lot of those peripherals require the MBED OS (higher RAM/ROM usage, but more "OS-like" functionality in return) to work, so moving them to this core would be a large challenge.

The alternative:  If you find something that works better here, why not submit PRs to the Arduino core to move the associated code over there?  We're all open source and always looking to improve.
